src/battle/battle_anim.c

```diff
@@ -13,13 +13,1855 @@
 #include "sprite.h"
 #include "task.h"
 #include "ewram.h"
+#include "graphics.h"
 
 // sprites start at 10000 and thus must be subtracted of 10000 to account for the true index.
 #define GET_TRUE_SPRITE_INDEX(i) (i - 10000)
 
 #define ANIM_SPRITE_INDEX_COUNT 8
 #define ANIM_ARGS_COUNT 8
 
+const struct OamData gOamData_837DF24 =
```

Comment:
Maybe all of this .rodata should get its own `src/data/battle_anim.h` file?

Reply:
The file is still under 5000 lines, so I think this is okay.

Reply:
I guess it's fine for now.  Having to scroll down 2k lines to reach the first function is a bit cumbersome, though.

Reply:
Yeah I had it in a separate header file first, but when I asked about it cam said to just put it in there, which is why I did